addons/website/static/src/xml/website_form.xml

```diff
@@ -16,6 +16,13 @@
         </span>
     </t>
 
+    <t t-name="website.s_website_form_status_custom_error">
+        <small class="s_website_form_custom_error me-2 small text-danger">
```

Comment:
Let's not break the consistency with other error template. refer this template with `id` instead `class`. :sweat_smile: 
```suggestion
        <small id="s_website_form_custom_error" class="me-2 small text-danger">
```

Reply:
When there will be multiple erroneous fields this tag will appear at multiple place, so there will be multiple tags with same id in DOM! apart from that to target all of them at once and apply css class is necessary.